.azure/templates/steps/prerequisites/install_java.yaml

```diff
@@ -1,12 +1,11 @@
-# Step to setup JAVA on the agent
+# Step to configure JAVA on the agent
 parameters:
   - name: JDK_VERSION
-    default: '11'
-
+    default: '17'
```

Comment:
aren't we still building bridge with Java 11 even if it can run on JVM 17?

Reply:
It is basically just the default version that is not used anywhere. But I changed it to 11.